docs/scripts.rst

```diff
@@ -130,6 +134,8 @@ Explanation for the keys:
 - ``urls``: dictionary of table names and the respective urls
 - ``version``: "1.0.0"
 
+``fields``: Although the Data retriever automatically detects the schema of the table, it is recommended to describe the schema of the table. This enables processing of the data faster since column detection increases the processing time.
```

Comment:
This line might be better fit under the section: **Full control over column names and data types**, possibly combined with the first few lines in that section, since they say pretty much the same thing.